OrbitService/OrbitService.cpp

```diff
@@ -16,6 +17,10 @@ OrbitService::OrbitService() {
 
   GTcpServer->Start(Capture::GCapturePort);
   ConnectionManager::Get().InitAsService();
+
+  core_app_ = std::make_unique<CoreApp>();
+  GCoreApp = core_app_.get();
+  GCoreApp->InitializeManagers();
```

Comment:
There is no app for the service mod. It might get confusing if some code is implicitly relying on some functionality from OrbitApp where it shouldn't. I think we would rather have null pointer dereference error in this case.

Are there many references to CoreApp? Should we just remove it? If the idea to rely messages to ui from OrbitCore code - we could use callbacks for it... WDYT?

Reply:
The original idea was to be able to trigger actions in OrbitApp from within OrbitCore, yes.  Now I think that CoreApp can be used a the main object that hold all managers.  

> "There is no app for the service mod."
Not sure I fully understand, but are you saying that there is no concept of "app" in the service?  I use "app" in the most generic way possible, I think it's ok for the service to have a CoreApp object.

I think there is definitely room for refactor, maybe we should dissociate the managers from the CoreApp?  Let's discuss it on another channel.